script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1468,32 +1573,37 @@ All for the sake of redeeming myself and her past―――
 [sha:bf1c38ef9f7e3625895a9b7cd369b1a8fceea114]{
 -- Page 479, Offset 34676.
 -- だから―――お願いします先輩。どうか、俺がロアに勝てる手助けをしてください」
-That's why―――I'm begging you, Senpai. Please help me find a way to defeat Roa."
+So in exchange――I'm begging you, Senpai.
+ Please help me find a way to win against Roa."
 }
 [sha:e3536a34a26b8468bb8acdd53b2cf82b9d1decba]{
 -- Page 480, Offset 34677.
 -- 「……フフ。大きくでましたね。
-"...Fufu. That's quite the tall order.
+"%{i}Ehehe%{/i}... That's quite the tall order.
 }
 [sha:6e27d5c86aeffcc0c4975b7f5563de9a31a2ab7f]{
 -- Page 480, Offset 34678.
 -- ロアを倒すではなくロアに勝利する、ですか。それは、今まで誰もできなかったコトですよ？」
-So instead of vanquishing Roa, you intend on triumphing over him, huh? You do realize that no one has ever done that before, right?"
+So you don't just want to vanquish Roa, you want to triumph over him, do you?
+ You do realize that nobody has ever managed that before, right?"
 }
 [sha:1b60bb9e77b3aa2ec72c0db900e6c55bb40c027a]{
 -- Page 481, Offset 34679.
 -- 「……勝算なしで大口を叩いてるのも承知の上です。でもそうしないともう先輩といられないんだから、やるしかないです」
-"I know that I'm running my mouth even though I have no chance of winning, but if that's the only way we can be together, then I have no other choice but to do it."
+"I know I'm talking a big game for someone with no chance of winning.
+ But if it's the only way we can be together, then that's just what I'll have to do."
 }
 [sha:78885dc29362142d9776475facad072a00abd3c5]{
 -- Page 482, Offset 34680.
 -- 「―――どんな無茶でも、ですか？」
-"―――Even if it's completely absurd?"
+"―――You mean you'll do whatever it takes?"
 }
 [sha:f9b3a9f7f1352ee2196ca95e4baa07ddb7b36b07]{
 -- Page 482, Offset 34681.
 -- 「はい。俺が死ぬ以外の手段があるなら、今はそれに懸けてみたい。……あ、いえ、そもそも選択肢があるかどうか、からですけど……」
-"Yeah. If there's any other way that doesn't involve dying, then I'd like to wager on that. Well, assuming there is another way in the first place..."
+"Yeah.
+ If there's a way that doesn't involve me dying, I'll wager everything I've got.
+ Though... well... I don't really know if there's a way to do it in the first place..."
```

Comment:
Suggestion doesn't really fit for me. Is the idea to push the "We should investigate that first" instead of just "I don't know if that's even an option"?

Reply:
"I want to give it my all! Actually... I'm not sure if there's something to give my all on..."

It is weirdly redundant, I agree (俺が死ぬ以外の手段があるなら), but uhh, yeah.

Reply:
It's a new line

Reply:
> Well, uh, assuming assuming such a method actually exists, that is
> Though, I guess we have to find out if such a method even exists first...
> Though, who knows if such a method even exists at all...

Reply:
Yeah, I dunno. Honestly I feel like I'm missing something about this line. The thing I wrote first seems to make the most sense in my head. 

Also, the response of Ciel needs to be changed if we change this line... Maybe just push it up until GPR. ahahah